firmware/include/config/secrets.h

```diff
@@ -1,39 +1,162 @@
 #pragma once
+
 /**
  * https://github.com/VIPnytt/Frekvens/wiki
  *
- * Definitions for the firmware sectron of the Frekvens project.
+ * Definitions for the firmware section of the Frekvens project.
```

Comment:
Typo fixes are always appreciated, so feel free to keep line, even if the rest of the file is going to be discarded. 👍